web-frontend/src/pages/TransferCoins/SendCoins/Advanced.tsx

```diff
@@ -52,12 +52,14 @@ function Modal({ ...props }) {
   function handlePresetGas() {
     setCustomFees(false);
     setPresetGasFees(true);
+    handleFees('1000');
+    setErrorAdvanced(null);
   }
 
   function handleCustomGas() {
     setCustomFees(true);
     setPresetGasFees(false);
-    handleFees('');
+    handleFees('0');
```

Comment:
why do you  want to set it at 0 ? this will put 0 in the input field

Reply:
i changed it to '' :)